pkg/pillar/nireconciler/linux_test.go

```diff
@@ -2133,14 +2141,15 @@ func TestIPv6LocalAndSwitchNIs(test *testing.T) {
 	t.Expect(recUpdate.UpdateType).To(Equal(nirec.NIReconcileStatusChanged))
 	t.Expect(recUpdate.NIStatus.Equal(niStatus)).To(BeTrue())
 
+	// Not running metadata server for IPv6 NI.
```

Comment:
This is based on the assumption that the NI is IPv4 or dual stack? Makes sense to put that in the comment.